src/main.rs

```diff
@@ -17,14 +19,15 @@ fn get_prompt_from_cli() -> String {
 }
 
 fn main() {
-    println!("Loading tokenizer...");
-    println!("Downloading from huggingface...");
-    println!("Loading model weights...");
-
-    let root = "../llama-7b-bytes";
-    let llama = load_on_disk(root);
     let dev: AutoDevice = Default::default();
-    let input_ids: Tensor<Rank2<1, 10>, usize, _> = dev.zeros();
+    let tokenizer = Tokenizer::from_file("../llama-7b-hf/tokenizer_config.json").unwrap();
```

Comment:
This doesn't work - what file do we have to pass in here? Or does llama require a more custom instantiation?